packages/oui-navbar/src/notification/navbar-notification.html

```diff
@@ -0,0 +1,119 @@
+<header class="oui-navbar-menu__header oui-navbar_mobile-only"
+    ng-if="::!!$ctrl.headerTitle">
+    <h3 class="oui-navbar-menu__title" ng-bind="::$ctrl.headerTitle"></h3>
+</header>
+<header class="oui-navbar-menu__header"
+    ng-if="::!!$ctrl.headerTemplate"
+    ng-bind-html="::$ctrl.headerTemplate">
+</header>
+<ul class="oui-navbar-menu__body oui-navbar-list">
+    <li class="oui-navbar-menu__item oui-navbar-notification"
+        ng-repeat="menuLink in ::$ctrl.menuLinks | limitTo: $ctrl.limitTo"
+        ng-class="!!menuLink.isActive ? 'oui-navbar-menu__item_active' : null">
+        <i class="oui-icon oui-icon_bicolor oui-navbar-notification__icon"
+            ng-class=":: {
+                'oui-icon_bicolor': menuLink.level !== 'incident',
+                'oui-icon-error_circle': menuLink.level === 'error',
+                'oui-icon-success_circle': menuLink.level === 'success',
+                'oui-icon-info_circle': menuLink.level === 'info',
+                'oui-icon-warning_circle': menuLink.level === 'warning',
+                'oui-icon-clock-wait': menuLink.level === 'incident'
+            }"
+            aria-hidden="true">
+        </i>
+
+        <!-- Link (href) -->
+        <a class="oui-navbar-notification__content"
+            ng-if="::!!menuLink.url"
+            ng-init="$ctrl.notificationsCreated = $last"
+            ng-click="menuLink.linkClicked(menuLink);$ctrl.navbarCtrl.toggleMenu()"
+            ng-href="{{::menuLink.url}}"
+            oui-navbar-group="{{::$ctrl.menuName}}"
+            oui-navbar-group-last="::$last && !$ctrl.footerTemplate && !$ctrl.footerTitle">
+            <div class="oui-navbar-notification__header">
+                <span class="oui-navbar-notification__title" ng-bind=":: menuLink.subject"></span>
+                <span class="oui-navbar-notification__time" ng-bind="menuLink.time"></span>
```

Comment:
No `::` on that one ?

Reply:
Not here, the purpose is to update the time when needed (eg. 26m ago, 2h ago, etc.).